src/components/SelectInput/SelectInput.jsx

```diff
@@ -326,6 +325,7 @@ const SelectInput = ({
   const Select = reactSelect.default;
   return (
     <div className="bootstrap-select-wrapper">
+      {label && <label>{label}</label>}
```

Comment:
```suggestion
      {label && <label for=xxxxxxx>{label}</label>}
```

la label non deve essere associata al campo con un l'attributo 'for' ? 

forse va anche messa la condizione per cui l'attributo 'for' viene messo solo se labelledby non è definito..

Reply:
è giusto, mi sono dimenticato di portare dentro questa info, su react è anche htmlFor.